visualization/app/codeCharta/util/dataMocks.ts

```diff
@@ -1745,8 +1745,8 @@ export const STATE: State = {
 		searchPattern: "",
 		margin: 48,
 		colorRange: {
-			from: 19,
-			to: 67,
+			from: 10,
+			to: 30,
```

Comment:
The test relied on the color set in the mock data, by adding mcc values to the other mock data i ensured that the new functionality is actually tested. The change in the "from" and "to" values has no deeper meaning, other than that i think smaller numbers in test cases are easier to comprehend

Reply:
If there's no need to change the values here, I would rather keep them to know what code changed exactly with this change.

If I understood correct, the test data is used in a way that it was not meant for originally and with the added entry, it'll work as expected? If that's the case, I would rather clone a full featured minimal map and add the necessary entries in the test that requires the changed data. Otherwise changed data could have impact on other tests without noticing. Especially, since the mocked data does not seem to represent actual input (missing values etc.).

Reply:
Fair enough, makes sense to me